orchagent/orch.h

```diff
@@ -218,6 +218,11 @@ class Orch
     static void recordTuple(Consumer &consumer, const swss::KeyOpFieldsValuesTuple &tuple);
 
     void dumpPendingTasks(std::vector<std::string> &ts);
+
+    /* Failure handling */
+    bool handleSaiCreateFailure(sai_status_t status);
```

Comment:
>bool [](start = 4, length = 4)

protected functions?
virtual functions?

Reply:
Made the function as protected virtual.